components/eam/src/chemistry/modal_aero/seasalt_model.F90

```diff
@@ -658,34 +515,19 @@ subroutine seasalt_emis(u10, u10cubed, lchnk, srf_temp, ocnfrc, ncol, cflx, emis
                 cflx_help2(:ncol) = 0.0_r8
                 if (Dg(i).ge.sst_sz_range_lo(nslt+m_om) .and. Dg(i).lt.sst_sz_range_hi(nslt+m_om)) then
                    cflx_help2(:ncol)=fi(:ncol,i)*ocnfrc(:ncol)*emis_scale
-                   if ( ( mixing_state == 0 ) .or. ( mixing_state == 2 ) ) then
-                      ! Mixing state 0: external mixture, replace mass with OM,
-                      !                 total number not modified
-                      ! Mixing state 2: internal mixture, replace mass with OM,
-                      !                 total number not modified
-                      cflx(:ncol,mn) = cflx(:ncol,mn) + cflx_help2(:ncol)*om_ssa(:ncol, i)
-                   else if ( ( mixing_state == 1 ) .or. ( mixing_state == 3 ) ) then
+
                       ! Mixing state 1: external mixture, add OM to mass and number
                       ! Mixing state 3: internal mixture, add OM to mass and number
                       cflx(:ncol,mn) = cflx(:ncol,mn) + cflx_help2(:ncol) * &
                                        (1._r8 / (1._r8 - om_ssa(:ncol, i)) - 1._r8)
-                   else
-                      ! Unknown mixing state assumption
-                      write(iulog, *) "Error: Unknown mixing_state value in seasalt_model.F90"
-                      exit
-                   end if
                 end if
```

Comment:
remove the spaces in `end if` and `end do`

Reply:
changed them to endif and enddo